docs(release): v0.11.0 release notes (#904)
flybywiresim/docs: docs/release-notes/v0110.md

|              [General Improvements](#general-improvements)              |

---

## Feature Details

### VNAV

We are thrilled to introduce the foundational layer of the vertical navigation capabilities (VNAV) in the A32NX. This release incorporates the rudimentary vertical prediction and 
guidance functions, laying the groundwork for advance functionalities in future updates.

Simulation pilots can expect the following in the current version:

- Precision in speed, altitude, time, and fuel predictions through all flight phases as showcased in the MCDU.
- Display of vertical predictions on the navigation screen including the top of descent, vertical path, target altitude, target speed, and relevant [Navigation Display symbology](#).
- Vertical guidance in managed modes providing vertical path and speed guidance. 
- Capability to set cruise steps directly in the MCDU.
- Set cruise steps directly in the MCDU. 

!!! info "Development Pipeline"
    We wanted to provide a short note that this first iteration release into the stable version does not include certain features that are still currently in development.

### flyPadOS Version 3.2 

> Our latest flyPadOS brings forward server key enhancements and fixes that pilots can now enjoy.

Reviewer: ```suggestion
Our latest flyPadOS brings forward serveral key enhancements and fixes that pilots can now enjoy.
```

!!! tip ""
    **EFB/ATSU Improvements** 

    We've not incorporated the reliable NOAA ([aviationweather.gov](https://aviationweather.gov/)) as a METAR source, ensuring you have the most accurate and up-to-date weather data. Additionally, our system will now utilize MSFS METAR data, moving away from the older FSX cloud data original provided through the FBW API. This will ensure a more synchronized weather experience in your flight deck.

    **EFB/FWC Update**

    Automatic Call Outs are now selectable in the flyPad. The basic configruation offered by Airbus is set as the default.

    **EFB/Payloads Enhancements**

    Pilots now have the capability for multi-gate boarding whenever multiple doors are open, streamlining the boarding process and saving time.

    You'll also receive a user-friendly UI message on the fuel/payload page when GSX is activated, ensuring clearer communication between the two systems.

    **EFB Functional Upgrades**

    We've added a few notable functional changes such as a "Pause at T/D" function. SimBridge connectivity issues have also been addressed.

>     Main dashboard and landing calculator have been refined to use your selected METAR source for a consistent weather information feed.

Reviewer 2: ```suggestion
    The main dashboard and landing calculator have been refined to use your selected METAR source for a consistent weather information feed.
```

    **EFB Cosmetic Enhancements**

>     We've rolled out aesthetic updates to the throttle configuration settings while also expanding support for multiple airframes. This ensures our flyPad is compatible with 
>     our future aircraft releases maintaining versatility between our products.

Reviewer 2: ```suggestion
    We've rolled out aesthetic updates to the throttle configuration settings, while also expanding support for multiple airframes. This ensures our flyPad is compatible with 
    our future aircraft releases, maintaining versatility between our products.
```

### Crossbleed Engine Starts

> Pilots are now able to initiate an engine start from crossbleed air. This procedure is normal performed when the APU Bleed start is not possible. For more information visit our 
> dedicated [Crossbleed Engine Start](#) page.

Reviewer 2: ```suggestion
Pilots are now able to initiate an engine start from crossbleed air. This procedure is normal performed when the APU Bleed start is not possible. For more information, visit our 
dedicated [Crossbleed Engine Start](#) page.
```

Author: self note to fix the hyperlink here

    **BMC and ECAM Updates**
    
    - Bleed air overpressure, underpressure, and overheat detection in the BMC.

    **Valve and Regulation Improvements**

    - Enhanced fan air valve regulation logic.
    - Overpressure valve implemented. 
    - Reworked high-pressure valve regulation.

    **Physics and Efficiency Updates**

    - Reworked engine compressor physics. 
    - Increased precooler cooling efficiency. 
    - Tuned APU bleed pressure.

    **Fault Light Logic**

    - Reworked pack fault light logic for better accuracy.

### Air Conditioning Simulation

This release introduces a realistically simulated intricate model of the Air Conditioning System tailored specifically to emulate an A320 equipped with an ACSC (Air 
Conditioning System Controller). Key additions include:

- ARINC Discrete Communication: To ensure high-fidelity data transfer and system controls.
- Valve Simulation and Dynamics: Detailed simulation of the TAPRV (Trim Air Pressure Regulating Valve) and TAV (Trim Air Valves) have been introduced. Along with this, users will observe a more realistic temperature response time from the pack, further enhancing the immersion until packs are fully modeled.

> - System Failures: A crucial aspect of any simulation is the unpredictability and the challenges it presents. Pilots can now set failures in various compontents. See our 

Reviewer 2: ```suggestion
- System Failures: A crucial aspect of any simulation is the unpredictability and the challenges it presents. Pilots can now set failures in various components. See our 
```

  [Custom Air Conditioning](#) page for more information.
  - Additional Failures Introduced: For those seeking an added layer of challenge and immersion, we've implemented scenarios like trim air system high-pressure malfunctions and duct overheats. 

### Flight Data Recorder Improvements

We have now included secondary engine parameters into the virtual Flight Data Recorder (FDR).

While most not be aware of this little fact, is that we do actually have an onboard FDR in the A32NX. This is a crucial component in the aircraft that records and logs various parameters and events. This data in real life is then used to analyze and investigate any incidents or accidents that may occur. 

> For the purposes of simulation we use it for providing technical support to our users and analyzing the same parameters and events. It allows us to troubleshoot issues like a drifting flight stick or if the aircraft was stalled by pilot error or a bug in the flight model.

Reviewer 2: ```suggestion
For the purposes of simulation, we use it for providing technical support to our users and analyzing the same parameters and events. It allows us to troubleshoot issues like a drifting flight stick or if the aircraft was stalled by pilot error or a bug in the flight model.
```

    **ISIS**

    A fix has been implemented for the ISIS alt tape, ensuring correct order representation for negative altitudes.

### General Improvements

The A32NX FMS has seen a host of updates and improvements. These include setting a default RNP of 0.3 for RNP AR procedure legs, adding basic support for FA legs, implementing the CHECK SPEED MODE message, vertical navigation functions, and more. 

Layout adjustments have been made to the PERF CLB, PERF CRZ, and PERF DES pages for better clarity. Additionally, the robustness of ILS selection has been improved, and procedures referencing runways are now loaded correctly. Other enhancements include displaying ILS identification and frequency on the ARRIVAL page, improved fuel warnings post-engine start, and appropriate use of station declination.

The MCDU has also seen multiple updates. This includes basic MCDU MENU functionality, annunciator support, and improvements in V-speed calculations. Remote MCDU protocol 

> enhancements include the transmission of annunciator light state, display, integral light brightness, and support for BRT/DIM keys. A few bugs, such as the FMGC annunciator light not illuminating and demanding QNH and TEMP readings when distant from the destination, have been addressed. The ZFW/ZFWCG auto-fill feature has been optimized to use desired values during the boarding process.

Reviewer 2: ```suggestion
enhancements include the transmission of annunciator light state, display, integral light brightness, and support for BRT/DIM keys. A few bugs, such as the FMGC annunciator light not illuminating and demanding QNH and TEMP readings when distant from the destination, have been addressed. The ZFW/ZFWCG autofill feature has been optimized to use desired values during the boarding process.
```

    !!! warning "**Navigraph + SimBrief Integration**"
        The authentication process for Navigraph services (SimBrief included) has changed. Please see the following page for more information:

        [SimBrief Integration](../fbw-a32nx/feature-guides/simbrief.md){.md-button}

    **Navigraph Airframe**

    The team at Navigraph have been kind enough to include our airframe as a selectable option on SimBrief. You no longer have to save our custom airframe to your account to 
    use it. If we decide to make any changes, it will be automatically updated in their system.

    [Guide Here](../fbw-a32nx/feature-guides/simbrief.md#simbrief-airframe){.md-button}

!!! warning "Important User Experience Changes"

    Please note the following changes.

    SimBridge is now required to use the following features:

    - Terrain on ND 
    - MCDU Remote Display
    - External Printer
    - Local Files feature
    - Company Routes

    [SimBridge Guides Here](../simbridge/index.md){.md-button}

    - Custom Flight Management System.
        - See the [Special Notes Section](../fbw-a32nx/feature-guides/cFMS.md#special-notes) on our custom FMS page for more details.

>     - Weather is inoperable as we wait for Asobo implementations.

Reviewer 3: Suggest changing "Weather is inoperable..." to "Weather radar is inoperable..."

>     - MSFS Built-in ATC and VFR maps are on limited support. [Guides Here](#)

Reviewer 3: Suggest changing "..are on limited support" to something like "...are partially supported" or "are supported on a limited basis" or something like that

Author: self note: realized there's no link here - fixed

    - [Throttle calibration](../fbw-a32nx/feature-guides/flypados3/throttle-calibration.md) is **mandatory**.

!!! info ""
    Downloads available through our [installer](../fbw-a32nx/installation.md).

    Please see our [Support Guide](../fbw-a32nx/support/index.md) and [Reported Issues](../fbw-a32nx/support/reported-issues.md).

---

## Quick Links

|                            Feature Shortcuts                            |
|:-----------------------------------------------------------------------:|
|                              [VNAV](#vnav)                              |
|                    [flyPadOS](#flypados-version-32)                     |
|          [Crossbleed Engine Starts](#crossbleed-engine-starts)          |
| [Air Conditioning Simulation + Failures](#air-conditioning-simulation)  |
| [Flight Data Recorder Improvements](#flight-data-recorder-improvements) |
|    [Flight Instrument Improvements](#flight-instrument-improvements)    |
|              [General Improvements](#general-improvements)              |

---

## Feature Details

### VNAV

> We are thrilled to introduce the foundational layer of the vertical navigation capabilities (VNAV) in the A32NX. This release incorporates the rudimentary vertical prediction and 

Reviewer 3: To me the use of the word "rudimentary" here sounds like something that is barely thought out or functional. How about using a word like "basic," which works well with the mention of "advanced" later in the same sentence, and I think is what is meant here?

> guidance functions, laying the groundwork for advance functionalities in future updates.

Reviewer 3: "advance" should be "advanced"

Simulation pilots can expect the following in the current version:

- Precision in speed, altitude, time, and fuel predictions through all flight phases.
- Display of vertical predictions on the navigation screen including the top of descent, vertical path, target altitude, target speed, and relevant [Navigation Display 
  symbology](../pilots-corner/advanced-guides/flight-guidance/vertical-guidance/nd-symbols.md).
- Vertical guidance in managed modes providing vertical path and speed guidance. 

> - Capability to set cruise steps directly in the MCDU.

Reviewer 3: What is the difference between this bullet (Capability to set cruise steps directly in the MCDU) and the one below it (Set cruise steps directly in the MCDU)?

- Set cruise steps directly in the MCDU. 

!!! info "Development Pipeline + Guides"
    We wanted to provide a short note that this first iteration release into the stable version does not include certain features that are still currently in development.

>     For an in-depth look at utilizing vertical guidance in the A32NX including what features are still in the pipeline see our dedicated advance guide below!

Reviewer 3: Suggest adding a comma between "A32NX" and "including"

    [Vertical Guidance Guides](../pilots-corner/advanced-guides/flight-planning/overview.md){.md-button}

### flyPadOS Version 3.2 

Our latest flyPadOS brings forward several key enhancements and fixes that pilots can enjoy.

!!! tip ""
    **EFB/ATSU Improvements** 

    We've incorporated the reliable NOAA ([aviationweather.gov](https://aviationweather.gov/)) as a METAR source, ensuring you have the most accurate and up-to-date weather 
    data. 

>     Additionally, our system can utilize MSFS METAR data, moving away from the older FSX cloud data original provided through the FBW API. This will ensure a more synchronized 

Reviewer 3: "original provided" should be "original**ly** provided"

    weather experience in your flight deck.

    **EFB/FWC Update**

    Automatic Call-Outs are selectable in the flyPad. The basic configuration offered by Airbus is set as the default.

    **EFB/Payloads Enhancements**

    Pilots have the capability for multi-gate boarding whenever multiple doors are open, streamlining the boarding process and saving time.

    !!! warning "Lack of Animation"

>         Please note that due to Asobo model issues the animation of the left aft door will not be visible. Use the EFB or lower ECAM to verify the door status of the aircraft.

Reviewer 3: Is it worth adding that we intend to rectify this in the future with our own model?

Author: Definitely -- added thanks

- Valve Simulation and Dynamics: Detailed simulation of the TAPRV (Trim Air Pressure Regulating Valve) and TAV (Trim Air Valves) have been introduced. Along with this, users will observe a more realistic temperature response time from the pack, further enhancing the immersion until packs are fully modeled.
- System Failures: A crucial aspect of any simulation is the unpredictability and the challenges it presents. Pilots can set failures in various components. See our 
  [Custom Air Conditioning](../fbw-a32nx/feature-guides/custom-air-conditioning.md) page for more information.
  - Additional Failures Introduced: For those seeking an added layer of challenge and immersion, we've implemented scenarios like trim air system high-pressure malfunctions and duct overheats. 

### Flight Data Recorder Improvements

The virtual Flight Data Recorder (FDR) now includes secondary engine parameters.

Many may not be aware, but the A32NX actually has an onboard FDR. This is a crucial component in the aircraft that records and logs various parameters and events. In real-life scenarios, this data is essential for analyzing and investigation incidents or accidents that may occur.

> For the purposes of simulation, our support teams use it for providing technical support to our users and analyzing the same parameters and events. It allows us to troubleshoot issues like a drifting flight stick or if the aircraft was stalled by pilot error or a bug in the flight model.

Reviewer 3: Don't think the stall example is the best since we don't encounter an actual stall very often. Also, it would be very unlikely to have been caused by a flight model issue. Suggest something more like, "or if abnormal aircraft or display behavior is caused by pilot error, a hardware issue, an MSFS issue, or a bug in our simulation." (In fact, you can probably replace the whole sentence after "issues like a" with this.)

Author: used your suggested and stripped sentence after "troubleshoot"
